src/vim-loader/object.ts

```diff
@@ -42,129 +64,126 @@ export class Object {
     this.instances = instances
     this._meshes = meshes
 
-    this.outlineAttribute = new ObjectAttribute(
+    this._outlineAttribute = new ObjectAttribute(
       false,
       'selected',
       'selected',
       meshes,
       (v) => (v ? 1 : 0)
     )
 
-    this.visibleAttribute = new ObjectAttribute(
+    this._visibleAttribute = new ObjectAttribute(
       true,
       'ignore',
       'ignore',
       meshes,
       (v) => (v ? 0 : 1)
     )
 
-    this.focusedAttribute = new ObjectAttribute(
+    this._focusedAttribute = new ObjectAttribute(
       false,
       'focused',
       'focused',
       meshes,
       (v) => (v ? 1 : 0)
     )
 
-    this.coloredAttribute = new ObjectAttribute(
+    this._coloredAttribute = new ObjectAttribute(
       false,
       'colored',
       'colored',
       meshes,
       (v) => (v ? 1 : 0)
     )
 
-    this.colorAttribute = new ColorAttribute(meshes, undefined, vim)
+    this._colorAttribute = new ColorAttribute(meshes, undefined, vim)
   }
-
-  private get meshBuilder () {
-    return this.vim.scene.builder.meshBuilder
-  }
-
+  
   /**
-   * Returns true if this object has geometry
+   * Checks if this object has associated geometry.
+   * @returns {boolean} True if this object has geometry, otherwise false.
    */
   get hasMesh () {
     return (this._meshes?.length ?? 0) > 0
   }
 
   /**
-   * Toggles selection outline for this object.
-   * @param value true to show object, false to hide object.
+   * Determines wether to render selection outline for this object or not.
    */
   get outline () {
-    return this.outlineAttribute.value
+    return this._outlineAttribute.value
   }
 
   set outline (value: boolean) {
-    if (this.outlineAttribute.apply(value)) {
-      if (value) this.vim.scene.addOutline()
-      else this.vim.scene.removeOutline()
+    if (this._outlineAttribute.apply(value)) {
+      if (value) this.vim._scene.addOutline()
+      else this.vim._scene.removeOutline()
     }
   }
 
   /**
-   * Toggles focused highlight for this object.
-   * @param value true to highlight object.
+   * Determines wether to render focus highlight for this object or not.
```

Comment:
Spelling: "whether" (worth doing a little search + replace to catch all misspelled instances)